examples/data-objects/inventory-app/webpack.config.cjs

```diff
@@ -38,26 +36,20 @@ module.exports = (env) => {
 			output: {
 				filename: "[name].bundle.js",
 				path: path.resolve(__dirname, "dist"),
-				library: "[name]",
-				// https://github.com/webpack/webpack/issues/5767
-				// https://github.com/webpack/webpack/issues/7939
-				devtoolNamespace: "fluid-experimental/tree-api",
```

Comment:
these bugs seem to be fixed, and removing this item doesn't seem to cause any issues as far as I can tell.